### DISCUSSION TITLE
WiinUPro not detecting Wii U Pro controller in Windows 11

### DISCUSSION THREAD

**Discussion by ceedveed** [KeyPuncher/WiinUPro#155] [Help]
Wii U Pro controller successfully pairs with Windows 11, but is not detected in the WiinUPro program. I've tried the same bluetooth dongle and Wii U Pro controller on Windows 10, and it works fine.

Has anyone got this to work under Windows 11? Any tips?

EDIT: I got it working. Install WiinUPro exe setup file. Uncheck Bluetooth driver when SCP installs. Make sure no Wii U Pro controller is in the Windows Bluetooth device list. Start WiinUpro. Press sync in WiinUPro. Press red sync button on back of Wii U Pro Controller. Now it gets connected and I can continue setting up.

I also do the following:

* After syncing, go to controller's configuration tab and configure to use the "ProController_XInput_layout.wup"
* Enable Vibration, then save as new profile. Configure to use that new profile as the default profile and enable Auto Connect.
* Enable Auto Add Xinput Devices in the Home tab.

Now when I want to play, I connect my Wii U Pro Controller to Windows 11 by pressing a button to activate the controller. Then I open up WiinUPro, and it will automatically connect my controller to the WiinUPro software, and also automatically add it as an Xinput device.

When I'm done playing, I close WiinUPro. For my 21H2 Windows 11 system, I turn bluetooth off and on. This is the only way I know how to turn off the Wii U Pro Controller. Holding down the power button seems to turn off the indicator light, but Windows 11 bluetooth settings will still show the controller connected.

The only issue is that I have to restart the PC sometimes if I closed WiinUPro, disabled BT then re-enabled BT.

EDIT2: I ended up keeping bluetooth off when I'm done with WiinUPro, and only turning it back on when I'm ready to use WiinUPro again. This helped solve the issue of me having to restart the PC.

**Comment by thanone2** [comment]
Hey! Thx man! this really helped!

**Comment by eshuaiii** [comment]
This is amazing, thank you! I followed this but ended up reinstalling SCP Drivers by selecting Force Driver Install using #80, this made XInput devices work and allowed me to use the controller on Steam!
